### PR TITLE
fix(tabs): export inset type and add inset as property

### PR DESCRIPTION
**Pull request by zeroedin**
## What I did

- Added inset as a decorated property 
- Exports InsetVariant type

### CODE REVIEW

File: elements/pfe-tabs/BaseTabs.ts [hunk 1]
```diff
@@ -74,6 +83,8 @@ export abstract class BaseTabs extends LitElement {
 
   id: string = this.id || getRandomId(this.localName);
 
+  @property({ reflect: true }) inset?: InsetVariant;
```
Comment (heyMP) [inline elements/pfe-tabs/BaseTabs.ts:86]: Do we need this property to be reflected to the attribute?

Comment (zeroedin) [reply elements/pfe-tabs/BaseTabs.ts]: After discussion in chat, you are correct.

File: elements/pfe-tabs/BaseTabs.ts [hunk 2]
```diff
@@ -152,9 +164,17 @@ export abstract class BaseTabs extends LitElement {
...
 
   override render() {
+    const classes = {
+      'inset-sm': this.inset === 'sm',
+      'inset-md': this.inset === 'md',
+      'inset-lg': this.inset === 'lg',
+      'inset-xl': this.inset === 'xl',
+      'inset-2xl': this.inset === '2xl',
+      'inset-page': this.inset === 'page'
+    };
```
Comment (heyMP) [inline elements/pfe-tabs/BaseTabs.ts:174]: We could shorthand this to:

```suggestion
    const classes = { [`inset-${this.inset}`]: true };
```

**Review by heyMP — APPROVED** [review]
Great use of classMap to solve this use case 👍

File: elements/pfe-tabs/BaseTabs.ts (hunk 2 above)
Comment (zeroedin) [reply elements/pfe-tabs/BaseTabs.ts:174]: I really phoned that one in at the end of the day huh? 😆

**Pull request merged**